### ISSUE TITLE
Mist w/ Ability To Ignore Alpha Channel (Transparent Sections) 

### ISSUE THREAD

**Issue by ndo360** [mist-project/mist-v2#21]
Greetings!

After multiple failed attempts I have managed to get the program to work! However a flaw has been uncovered... Currently during the Misting process of any image converts the alpha channel / transparent background into pure black.

For creatives to be able to use their misted images in various mediums, maintaining any and all transparent elements is a huge need for most misted images to be usable in projects. Not only would it still protect the entire photo, but the transparent sections would remain transparent enabling easier use for those that have images with transparent elements. (As filling in the transparent layer with pure black could cause issues when attempting to remove those sections, thus degrade the image and not having it be as clean.)

Is this possible? The program works great, just needs a tweak to account for transparency to enable smooth usage of their own misted images. I guess a "Smart Mist / Selective Mist / Targeted Mist" mode more or less?

**Comment by caradryanl** [comment]
Yes I have seen your problem through Discord. We will update this features in two weeks.
